models/workspacemanager/mantid_workspace_provider.py

```diff
@@ -42,3 +42,9 @@ def get_workspace_handle(self, workspace_name):
         if isinstance(workspace_name, Workspace):
             return workspace_name
         return AnalysisDataService[workspace_name]
+
+    def get_workspace_name(self, workspace):
```

Comment:
That must have slipped through. The problem with Python and not having a compiler!